config_v1.json

```diff
@@ -1,4 +1,4 @@
 {
   "packageName": "jcapiv1",
-  "packageVersion": "3.0.0"
+  "packageVersion": "3.1.0"
```

Comment:
Why is the package version 3.1.0 for Python, but 2.2.0 for Go and Ruby?

Reply:
It's based on the release version, but I'm not sure why/when they diverged.